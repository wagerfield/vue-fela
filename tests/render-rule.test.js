import { createRenderer } from 'fela'
import { renderRule } from '../index'
import rules from './component-rules'

const fela = createRenderer()
const rule = rules.border
const props = {
  outerRadius: 0,
  innerRadius: 0,
  active: false
}

describe('renderRule(renderer, rule, props)', () => {

  beforeEach(fela.clear)

  it('returns a string', () => {
    expect(renderRule(fela, rule, props)).toEqual(expect.any(String))
  })

  it('renders expected snapshot', () => {
    expect(renderRule(fela, rule, props)).toMatchSnapshot()
  })
})
