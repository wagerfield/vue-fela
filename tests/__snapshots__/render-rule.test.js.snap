// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderRule(renderer, rule, props) renders expected snapshot 1`] = `"a b c"`;
